src/sentry/integrations/jira/notify_action.py

```diff
@@ -16,39 +16,43 @@
 from sentry.utils.http import absolute_uri
 from sentry.web.decorators import transaction_start
 
+from .utils import get_name_for_jira
```

Comment:
tiniest nit - you can import this along with the other things from that file on lines 9 - 11